extensions/wikia/AdEngine/js/context/adContext.js

```diff
@@ -42,13 +42,6 @@ define('ext.wikia.adEngine.adContext', [
 		return context.targeting.pageType === pageType;
 	}
 
-	function isRecoveryModuleEnabled(wgCountries, contextVariable) {
-		var isGeoSupported = geo.isProperGeo(wgCountries),
-			isNotDisabledOnWiki = contextVariable !== false;
-
-		return !!(isNotDisabledOnWiki && isGeoSupported);
-	}
-
 	function setContext(newContext) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Function has a complexity of 53 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)